src/frontend/citizen-portal/src/app/components/update-dispute-landing/update-dispute-landing.component.ts

```diff
@@ -61,6 +61,6 @@ export class UpdateDisputeLandingComponent implements OnInit {
   }
 
   goToUpdateDisputeContact(): void {
-    this.disputeService.goToUpdateDisputeContact(this.state.params);
+    this.disputeService.goToUpdateDisputeContact(this.state.params, this.isEmailVerified === null ? false : true);
```

Comment:
`this.disputeService.goToUpdateDisputeContact(this.state.params, this.isEmailVerified !== null);`

Reply:
done